tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py

```diff
@@ -29,12 +29,14 @@ def test_notification_bar(self):
 
         utility_tray.wait_for_notification_container_displayed()
 
-        # Assert there is one notification is listed in notifications-container
+        # Assert there is a system notification and is listed in notifications-container
         notifications = utility_tray.notifications
-        self.assertEqual(1, len(notifications), 'Expected one notification.')
-
-        # Assert notification is listed in notifications-container
-        self.assertEqual(self._notification_body, notifications[0].content)
```

Comment:
Same here, notifications[0] must have come from the system app, so I could use this check still and not the stuff that I added down here.
Also, I might want to add a check regarding len(notifications) becoming 1 larger than before (not removing the one check above, but replacing it with a more fine-grained check).